### ISSUE TITLE
Database migration to include project managers on project db entry

### ISSUE THREAD

**Issue by jbubar** [hackforla/VRMS#1386]
### Overview
We want to display the PM's for the project on the Project page, and the first step is having that information in the backend. 

Currently the project manager information (who can change project meeting times) are being kept in the [User model ](https://github.com/hackforla/VRMS/blob/development/backend/models/user.model.js)(managedProjects array of project IDs) however, it is not in the project model. What we need is to add the User ids into the managedByUsers array on the [project model](https://github.com/hackforla/VRMS/blob/development/backend/models/project.model.js).

### Action Items

This should be done similarly to what @dannyprikaz is working on via a `python notebook` of some sort.

 - [ ] Add a field to the projects model to keep track of the managers
 - [ ] Do a database migration to add this new field to the test db.
 - [ ] Create a new db query that adds a specified user id to a project
 - [ ] Create a new db query that removes a specified user id from a project
    * These queries may need their own api route
    * These queries should be PUT requests because they're updating a current entrey
  * Note from Trillium that I don't have enough of a mental model of the project right now to point out where in the code this needs to go. Will update.
 - [ ] Create unit tests for all new functionality / update existing unit tests for any changed functionality

### Resources/Instructions

- This issue is part of the Epic #1986

**Comment by jbubar** [comment]
Do we still want to do this issue? @Spiteless.

**Comment by juliagab56** [comment]
This issue needs to be cleaned up and possibly broken down into two (based on the multiple labels) before we can assign a milestone or move it forward on the board @JackHaeg @jbubar @Spiteless @ExperimentsInHonesty

**Comment by JackHaeg** [comment]
@jng34 This is the next issue in the epic that we discussed on tonight's All team call. Please feel free to assign this issue to yourself and change the status to "In Progress" once you start working on it 🙂

**Comment by JackHaeg** [comment]
- Closing as duplicate of #1987
